### ISSUE TITLE
Add homebrew tap for OSX

### ISSUE THREAD

**Issue by dennisdegreef** [ellotheth/pipethis#12]
It would be nice to be able to install pipethis with Homebrew

**Comment by dennisdegreef** [comment]
Fixed with https://github.com/dennisdegreef/homebrew-pipethis

I just need to add a `README.md` and `.travis.yml` to that repository on how to install.

```
$ brew tap dennisdegreef/pipethis
==> Tapping dennisdegreef/pipethis
Cloning into '/usr/local/Library/Taps/dennisdegreef/homebrew-pipethis'...
remote: Counting objects: 4, done.
remote: Compressing objects: 100% (2/2), done.
remote: Total 4 (delta 0), reused 4 (delta 0), pack-reused 0
Unpacking objects: 100% (4/4), done.
Checking connectivity... done.
Tapped 1 formula (26 files, 19.9K)
```

```
$ brew info pipethis
dennisdegreef/pipethis/pipethis: stable 64
Replace your `curl | sh` installers
https://github.com/ellotheth/pipethis
Not installed
From: https://github.com/dennisdegreef/homebrew-pipethis/blob/master/Formula/pipethis.rb
```

```
$ brew install pipethis
==> Installing pipethis from dennisdegreef/pipethis
==> Downloading https://github.com/ellotheth/pipethis/releases/download/v0.5.1/pipethis-v0.5.1-darwin-amd64.tar.bz2
==> Downloading from https://github-cloud.s3.amazonaws.com/releases/50157134/...
######################################################################## 100.0%
🍺  /usr/local/Cellar/pipethis/64: 2 files, 7.0M, built in 5 seconds
```

```
$ which pipethis
/usr/local/bin/pipethis
```

```
$ ls -l $(which pipethis);
lrwxr-xr-x  1 dennisdegreef  admin  34 Jun 18 21:20 /usr/local/bin/pipethis -> ../Cellar/pipethis/64/bin/pipethis
```

```
$ pipethis -version
2016/06/18 21:26:33 pipethis v0.5.1-darwin-amd64 (go1.6.2)
```

**Comment by dennisdegreef** [comment]
The version is not detected properly yet, fixed that in https://github.com/dennisdegreef/homebrew-pipethis/commit/4ddc171bce9a1029c288206637d353a05c573942

```
$ brew remove pipethis
Uninstalling /usr/local/Cellar/pipethis/64... (2 files, 7.0M)
```
```
$ brew install pipethis
==> Installing pipethis from dennisdegreef/pipethis
==> Downloading https://github.com/ellotheth/pipethis/releases/download/v0.5.1/pipethis-v0.5.1-darwin-amd64.tar.bz2
==> Downloading from https://github-cloud.s3.amazonaws.com/releases/...
######################################################################## 100.0%
🍺  /usr/local/Cellar/pipethis/0.5.1: 2 files, 7.0M, built in 4 seconds
```
```
$ brew info pipethis
dennisdegreef/pipethis/pipethis: stable 0.5.1
Replace your `curl | sh` installers
https://github.com/ellotheth/pipethis
/usr/local/Cellar/pipethis/0.5.1 (2 files, 7.0M) *
  Built from source on 2016-06-18 at 21:48:03
From: https://github.com/dennisdegreef/homebrew-pipethis/blob/master/Formula/pipethis.rb
```

**Comment by ellotheth** [comment]
Fixed in #13
